doc/en/faq/performance.rst

```diff
@@ -10,53 +10,44 @@ This is a guide on improving Centreon's performance
 Databases
 *********
 
-The database server is one of the central components of Centreon. Its
-performance has a direct impact on the end user application's speed. Centreon
-uses two or three databases depending on your monitoring broker:
-
- * ``centreon`` -- Storing metadata
- * ``centreon_storage`` -- Real-time monitoring and history
-
+The database server is one of the central components of Centreon. Its performance has a direct impact on the end user
+application's speed. Centreon uses two databases:
 
+ * **centreon** : Storing metadata and configuration
+ * **centreon_storage** : Real-time monitoring and history
 
 Indexes
 =======
 
-Databases use indexes to speed up queries. In case indexes are missing queries
-are executed slower.
+Databases use indexes to speed up queries. In case indexes are missing queries are executed slower.
 
 .. _synchronizing-indexes:
 
 Synchronizing indexes
 *********************
 
-Starting with Centreon ``2.4.0`` for each release, index information files are
-generated. They are found in ``data`` folder usually located next to the
-``bin`` or ``www`` folders. They are JSON files and there is one for each database:
+For each release, index information files are generated. They are found in **/usr/share/centreon/bin/data** folder.
+They are JSON files and there is one for each database:
 
- * ``centreonIndexes.json`` -- Indexes for ``centreon`` database
- * ``centreonStorageIndexes.json`` -- Indexes for ``centreon_storage`` database
- * ``centreonStatusIndexes.json`` -- Indexes for ``centreon_status`` database
+ * **centreonIndexes.json**: Indexes for **centreon** database
+ * **centreonStorageIndexes.json**: Indexes for **centreon_storage** database
 
-Check if your database is desynchronized::
+Check if your database is desynchronized: ::
 
-  $ cd CENTREONBINDIR
-  $ ./import-mysql-indexes -d centreon -i ../data/centreonIndexes.json
+    $ cd /usr/share/centreon/bin
+    $ /opt/rh/rh-php71/root/bin/php import-mysql-indexes -d centreon -u root -i data/centreonIndexes.json
```

Comment:
just a comment
```suggestion
    $ scl enable rh-fpm71 bash
    $ php/php import-mysql-indexes -d centreon -u root -i data/centreonIndexes.json
```

Reply:
We don't know if another software using PHP is present on serveur of no.
More secure to use direct access